templates/certified/index.html

```diff
@@ -6,250 +6,632 @@
-      <a href="https://canonical.com/partners/become-a-partner">Learn more</a>
+  <div class="row">
+    <div class="col-small-1 col-medium-1 col-1">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/0f4ef2b8-adlink-logo.png",
+        alt="Adlink",
+        attrs={"class":"p-logo-section__logo"},
+        width="280",
+        height="280",
+        hi_def=False,
+        loading="lazy",
+        ) | safe
+      }}
+    </div>
+    <div class="col-small-1 col-medium-1 col-1">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/4e5c6776-hp-enterprise-logo.png",
+        alt="Hewlett Packard Enterprise",
+        attrs={"class":"p-logo-section__logo"},
+        width="280",
+        height="280",
+        hi_def=False,
+        loading="lazy",
+        ) | safe
+      }}
+    </div>
+    <div class="col-small-1 col-medium-1 col-1">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/0f36dbee-Inspur.svg",
+        alt="Inspur",
+        attrs={"class":"p-logo-section__logo"},
+        width="280",
+        height="280",
+        hi_def=False,
+        loading="lazy",
+        ) | safe
+      }}
+    </div>
+    <div class="col-small-1 col-medium-1 col-1">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/1d02aa46-aaeon-logo.png",
+        alt="Aaeon",
+        attrs={"class":"p-logo-section__logo"},
+        width="280",
+        height="280",
+        hi_def=False,
+        loading="lazy",
+        ) | safe
+      }}
+    </div>
+    <div class="col-small-1 col-medium-1 col-1">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/8fdc3ebf-raspberry-pi-logo.png",
+        alt="Rapsberry Pi",
+        attrs={"class":"p-logo-section__logo"},
+        width="280",
+        height="280",
+        hi_def=False,
+        loading="lazy",
+        ) | safe
+      }}
+    </div>
+    <div class="col-small-1 col-medium-1 col-1">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/2b253402-advantech-logo.png",
+        alt="Advantech",
+        attrs={"class":"p-logo-section__logo"},
+        width="280",
+        height="280",
+        hi_def=False,
+        loading="lazy",
+        ) | safe
+      }}
+    </div>
+    <div class="col-small-1 col-medium-1 col-1">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/d94e3a65-asrock.svg",
+        alt="ASRock",
+        attrs={"class":"p-logo-section__logo"},
+        width="280",
+        height="280",
+        hi_def=False,
+        loading="lazy",
+        ) | safe
+      }}
     </div>
-    <div class="col-5 u-hide--medium u-hide--small u-align--center">
+  </div>
+</section>  
+
+<section class="p-strip--suru-shape-dark is-deep">
+  <hr class="is-fixed-width is-dark">
+  <div class="row">
+    <div class="col-3 col-small-1 col-medium-1">
```

Comment:
This might be unnecessary but I see a little bit of padding at the top of the image in design
<img width="728" alt="Screenshot 2022-12-13 at 9 52 32 am" src="https://user-images.githubusercontent.com/57550290/207286056-43d3c803-48a9-4be5-bea4-4a42773bddff.png">

Reply:
good spot - updated!